src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -493,6 +493,7 @@ public void setUp() {
                 .withSectionIndex(2)
                 .add();
 
+        vlnew2.newLoad().setId("LOAD_WITH_NULL_NAME").setBus("NNEW2").setConnectableBus("NNEW2").setP0(600.0).setQ0(200.0).setName(null).add();
```

Comment:
Add a test with id and name not null and different

Reply:
i add a test with name not null
but what do you mean different ?

Reply:
id and name different